containers/EmpSponsors.ts

```diff
@@ -72,8 +75,12 @@ const useEmpSponsors = () => {
   // We set the poll interval to a very slow 5 seconds for now since the position states
   // are not expected to change much.
   // Source: https://www.apollographql.com/docs/react/data/queries/#polling
+  let clientName = "UMA";
+  if (network !== null) {
+    clientName += network.chainId.toString();
```

Comment:
nit: could you not make this a const and ditch the line 78 with:
```
const clientName = `UMA${network.chainId.toString()}
```

Reply:
Yeah this is doable because if the `clientName !== UMA42` it will default to the mainnet URL. I think node.js calculates `${undefined}` as `undefined` though so it would just return `UMAundefined` which is OK.